nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-2-6-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -81,3 +81,8 @@ The following binary components are provided under the Apache Software License v
       in some artifacts (usually source distributions); but is always available
       from the source code management (SCM) system project uses.
 
+  (ASLv2) aws-msk-iam-auth
+    The following NOTICE information applies:
+      aws-msk-iam-auth
+      Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
`aws-msk-iam-auth` pulls in a couple of other dependencies transitively (e.g. further aws libraries, netty).
The `NOTICE` file needs to be updated with those as well.

Reply:
Thanks for noticing it! Will update soon.